src/api/app/controllers/webui/packages/bs_requests_controller.rb

```diff
@@ -9,9 +9,6 @@ def index
         requests_query = BsRequest::DataTable::FindForPackageQuery.new(@project, @package, parsed_params)
         @requests_data_table = BsRequest::DataTable::Table.new(requests_query, params[:draw])
 
-        # NOTE: This session is used by requests/show
```

Comment:
Why did you remove this feature in the current interface?

Reply:
it was agreed with @coolo and @bgeuken, its not being used, not being tested and not full implemented. so its gone. @DavidKang

Reply:
please read the whole comments @DavidKang  https://github.com/openSUSE/open-build-service/pull/6851#pullrequestreview-196774356

Reply:
@vpereira, I read this comment before, but this not explain the reason for your decision to remove this feature. BTW, I agree if it's not used we can remove it.

Reply:
To give some background here: I implemented this feature to help me going through a lot of project requests. I also explained it to @bgeuken who extended it (with tests), but the feature still doesn't really work relying on sessions, which makes working in multiple tabs quite adventures and confusing. 

We don't know if others are using it, but we know the user story itself is there - but not too important to keep.

Reply:
@coolo, thanks for the clarification.

Reply:
And I just verified with @DimStar77 - he didn't know about this feature :)

Reply:
Please add the reason why this is removed in the commit message. It is important information you can not deduce from the code. In general a description in the commit messages is a good idea :wink: Remember to include **why** the changes are needed :smile: